apps/commudle-admin/src/app/feature-modules/email-confirmations/components/user-role-confirmation/user-role-confirmation.component.ts

```diff
@@ -19,11 +21,13 @@ export class UserRoleConfirmationComponent implements OnInit, OnDestroy {
   event: IEvent;
   communityGroup: ICommunityGroup;
   EUserRoles = EUserRoles;
+  acceptRole = false;
```

Comment:
Let's write data type!

Reply:
Type boolean trivially inferred from a boolean literal, remove type annotation.

It shows this error